wild/tests/sources/non_string_merging1.s

```diff
@@ -0,0 +1,43 @@
+.section .rodata.foo, "aM", @progbits, 1
+.align 1
+
+.globl s1h
+s1h: .ascii "Hello"
```

Comment:
It could be good to put some null bytes within one of the strings, just to verify that this works correctly.